### DISCUSSION TITLE
“vAmiga” can’t be opened because Apple cannot check it for malicious software.

### DISCUSSION THREAD

**Discussion by Panthros** [dirkwhoffmann/vAmiga#658] [General]
I am assuming this error is due to not having signed the application with a certificate?  I love your C64 and Amiga emulators.  Have you looked into the cost to sign your applications?  Perhaps a Patreon or something that people could donate to help you get that if you do not have the means?  Thank you for all you do to support Mac users and keeping Commodore 64 and Amiga alive today!

**Comment by dirkwhoffmann** [comment]
> I am assuming this error is due to not having signed the application with a certificate?

Yes, exactly. I do agree that this is an annoying error message. I'll think about signing up to the developer program in future which is the only way to get rid of it. I haven't done it yet because I didn't think it was worth paying $99 USD per year for having a message not appear.

**Reply by nekoniaow** [reply]
As a user I would gladly pay for the open source apps I use rather than have no guarantees about their security.
